t/testrules.yml

```diff
@@ -14,6 +14,7 @@ seq:
     - xt/40-dbsetup.t
     - xt/45*.t
```

Comment:
Is there a reason not to add 45-48 to the "par" (parallel) section below?

Reply:
Not that I'm aware of. I'll PR and we'll see what breaks.